src/components/help/HelpContainer.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { ReactComponent as ExitSVG } from 'media/icons/times-light.svg';
+import { ReactComponent as QuestionCircleSVG } from 'media/icons/question-circle-solid.svg';
+import HelpPanel from './HelpPanel';
+
+const styles = (theme) => ({
+  panelContainer: {
+    color: '#333F52',
+    fontFamily: theme.typography.fontFamily,
+    right: '0',
+    'max-width': '815px',
+    height: 'calc(100% - 64px)', // 64px = height of top nav bar
+    position: 'fixed',
+    'background-color': '#E9ECEF',
+    display: 'flex',
+    'flex-direction': 'column',
+    'flex-wrap': 'nowrap',
+    'z-index': 1,
+  },
+  anotherContainer: {
+    position: 'relative',
+    height: '100%',
+  },
+  background: {
+    background: '#000000',
+    left: 0,
+    right: 0,
+    opacity: 0.4,
+    height: '100%',
+    position: 'fixed',
+    'z-index': 1,
+  },
+  close: {
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    padding: '1em',
+    width: '3em',
+    height: '3em',
+    cursor: 'pointer',
+    ...theme.mixins.jadeLink,
+  },
+  footer: {
+    position: 'absolute',
+    bottom: '0',
+    background: '#d7dbdf',
+    width: '100%',
+    height: '100px',
+    'padding-left': '30px',
+    'padding-top': '15px',
+    'padding-bottom': '15px',
+  },
+  jadeLink: {
+    ...theme.mixins.jadeLink,
+  },
+  helpOpen: {
+    height: '48px',
+    width: '48px',
+    position: 'absolute',
+    right: 0,
+    ...theme.mixins.jadeLink,
+  },
+  helpIcon: {
+    padding: '8px',
```

Comment:
You need:
```
width: '100%',
height: '100%'
```

to fix the issue in FF.  This makes it take the 48px that you set above.

Also, can you add ```cursor: 'pointer'``` here too?